gdx/src/com/badlogic/gdx/utils/Json.java

```diff
@@ -572,6 +572,26 @@ public void writeValue (Object value, Class knownType, Class elementType) {
 				writeObjectEnd();
 				return;
 			}
+			if (value instanceof ObjectIntMap) {
```

Comment:
Why there are so much if blocks and no else if?
it always checks every if block if it matches.

Reply:
It does not, because each returns early. I didn't write this style, I was matching the existing code.

Reply:
@tommyettinger Yes, but i was wondering in major thinking i was wondering about that style. All right could leave that.